### DISCUSSION TITLE
Türen als Update

### DISCUSSION THREAD

**Discussion by FlorianHeck2** [FlorianHeck2/codespaces-jupyter#6] [Announcements]
Hey Zusammen,
Diese Woche habe ich als Update Ein [Ziel](https://github.com/FlorianHeck2/codespaces-jupyter/blob/4e1401ab08917e7c36429a1a3d3ca8a520bf5cd8/project/main.py#L39C1-L39C9) in mein Spiel eingebaut und die [Türen](https://github.com/FlorianHeck2/codespaces-jupyter/blob/4e1401ab08917e7c36429a1a3d3ca8a520bf5cd8/project/main.py#L99) verbessert.
Schaut gerne mal rein und probiert mal aus, was schon so geht. (kleiner Tipp: die Türe öffnet mit E, das Ziel muss man suchen)

**Comment by Flopfy** [comment]
Sieht sehr Interessant aus. Von deinem Code kann ich einiges lernen:) Gefällt mir.

**Reply by FlorianHeck2** [reply]
freut mich zu hören, danke :)

**Comment by laraanastasia** [comment]
looks great Flo!

**Reply by FlorianHeck2** [reply]
thanks

**Comment by NicoOblaender** [comment]
Jeah, TÜREN. Ich freu mich so auf das ganze Spiel

**Comment by LeoEnglert** [comment]
<img width="246" alt="Bildschirmfoto 2024-01-25 um 18 26 10" src="https://github.com/FlorianHeck2/codespaces-jupyter/assets/63335113/65d185d4-3183-45f3-aa27-cf23c0777032">

Bin ich hier richtig?
